### PR TITLE
Improvement

### PR DESCRIPTION
**Pull request by sirpa44**

### CODE REVIEW

File: Api/.htaccess [hunk 1]
```diff
@@ -0,0 +1,4 @@
+AuthName "Protected"
+AuthType Basic
+AuthUserFile "/var/www/patpat.com/public_html/test/apiREST/.htpasswd"
```
Comment (siwane) [inline Api/.htaccess]: This file does not exist and it is bad practice to authenticate app via htaccess

File: Api/.htpasswd [hunk 2]
```diff
@@ -0,0 +1 @@
+antoine:$1$XCW5Blax$ON4r5XhNBj6yNWigN0tBN0
```
Comment (siwane) [inline Api/.htpasswd]: Remove this file if not used

File: Api/app/App.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?php
```
Comment (siwane) [inline Api/app/App.php]: Remove it

File: Api/model/Csv.php [hunk 4]
```diff
@@ -41,6 +40,7 @@ public function getOne($setting)
      */
     private function contentExtract()
     {
+//        var_dump($this->path);die();
```
Comment (siwane) [inline Api/model/Csv.php]: Please remove unusefull commented code

File: composer.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+        "email": "sirpa.antoine@gmail.com"
+      }
+    ],
+    "require": {
+      "psr/http-message": "^1.0"
+    },
+
+    "autoload": {
+      "psr-4": {
+        "Api\\":"Api"
```
Comment (siwane) [inline composer.json]: Talk to urself

File: Api/.htaccess (hunk 1 above)
Comment (siwane) [reply Api/.htaccess]: Please use ur htaccess to protect directory against direct call (redirect all http calls to index.php)

**Review by siwane — CHANGES_REQUESTED** [review]

File: Api/.htpasswd (hunk 2 above)
Comment (sirpa44) [reply Api/.htpasswd]: done

File: Api/app/App.php (hunk 3 above)
Comment (sirpa44) [reply Api/app/App.php]: done

File: Api/model/Csv.php (hunk 4 above)
Comment (sirpa44) [reply Api/model/Csv.php]: done

File: composer.json (hunk 5 above)
Comment (sirpa44) [reply composer.json]: done

File: Src/core/controller/PostController.php [hunk 6]
```diff
@@ -1,8 +1,8 @@
 <?php
-namespace core\controller;
+namespace Aot\core\controller;
```
Comment (siwane) [inline Src/core/controller/PostController.php]: `core` keyword is a folder where components are generic. It means they can be reuse for another project (App, Router...). Can u remove this folder from path please?

**Review by siwane — CHANGES_REQUESTED** [review]

File: Src/core/controller/PostController.php (hunk 6 above)
Comment (sirpa44) [reply Src/core/controller/PostController.php]: done

**Review by siwane — APPROVED** [review]

**Review by siwane — CHANGES_REQUESTED** [review]
The project seems to not work, maybe a problem of autoloading?

**Pull request closed without merging**